sdk/schemaregistry/schema-registry-json/review/schema-registry-avro.api.md

```diff
@@ -1,42 +0,0 @@
-## API Report File for "@azure/schema-registry-json"
```

Comment:
is it expected that we're removing all public surface from this package? is the package not needed anymore or does this file need to be regenerated?

Reply:
When the package was first made, this API View file was copied from the schema-registry-avro, hence the title. We have a newly generated API Report for the schema-registry-json now.

Reply:
ah so this is just cleanup. Thanks for explaining!